### PR TITLE
Few very minor architecture nits

### PR DESCRIPTION
**Pull request by MaxF12**

### CODE REVIEW

File: draft-ietf-taps-arch.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ The traditional model of using sockets for networking can be represented as foll
   (typically implemented in the system's kernel).
 - TCP and UDP in the kernel send and receive data over the available network
   layer interfaces.
-- Sockets are bound directly to transport-layer and network-layer addresses,
+- Sockets are bound directly to transport layer and network layer addresses,
```
Comment (tfpauly) [inline draft-ietf-taps-arch.md]: Why no hyphens here?

Comment (MaxF12) [reply draft-ietf-taps-arch.md]: It was just for consistentcy, some times hyphend were used and sometimes they were not. I am used to the version without them, if its more common to use them I can change it to that.

Comment (tfpauly) [reply draft-ietf-taps-arch.md]: Cases where we say "the transport layer" should not be hyphenated, which is the case. However, if you use the term as an adjective, such as "a transport-layer address", you need to add a hyphen.

The one error I see in the document by those rules is just that "available network layer interfaces" should be "available network-layer interfaces".

Comment (MaxF12) [reply draft-ietf-taps-arch.md]: Oh of course, sorry my bad. Thanks for the explanation, fixed!

**Pull request merged**